# no-modulo-one

Modulo by 1 always returns 0. This may be a typo or a logic issue.

## Rule Details

Examples of **incorrect** code for this rule:

```js
x % 1
```

Examples of **correct** code for this rule:

```js
x % 2
```
